docs/CRSF.md

```diff
@@ -40,36 +46,43 @@ Configuration of MAVLink stream rates
 - SRx_POSITION = 2
 - SRx_RAW_SENS = 0 (for most of you this one is unimportant, keep it at 0 unless you really need it)
 
-Configuration for CRSF receiver
+Configuration for CRSF receiver:
 
-Setting up ArduPilot for a CRSF receiver can be a bit tricky, as it can depend on the used board, and might need BRD_ALT_CONFIG to be set to a specific value. It is best to consult the ArduPilot wiki, or ask in the ArduPilot discussion channel.
+Setting up ArduPilot for a CRSF receiver can be a bit tricky, as it depends on the flight controller board, and might need BRD_ALT_CONFIG to be set to a specific value. It is best to consult the ArduPilot wiki, or ask in the ArduPilot discussion channel.
 
 For my Matek H743 board the configuration is:
 
 - BRD_ALT_CONFIG = 1
 - RC_PROTOCOLS = 536 or 512
-- SERIAL7_BAUD = irrelevant
+- SERIAL7_BAUD = irrelevant (baud rate is determined by ArduPilot)
 - SERIAL7_OPTIONS = 0
 - SERIAL7_PROTOCOL = 23
 
+Notes:
+- There are more RC options available in the 'RC_OPTIONS' parameter. ([ArduPilot Docs for RC_OPTIONS](https://ardupilot.org/plane/docs/parameters.html#rc-options-rc-options)) 
+- RSSI_TYPE should be set to either 3 or 5. ([ArduPilot Docs for RSSI_TYPE](https://ardupilot.org/plane/docs/parameters.html#rssi-type-rssi-type)) 
+
 
 ## mLRS Rx Module Setup
 
-These configurations are not strictly neccesary, but recommended.
+These configurations are not strictly neccesary, but recommended for ArduPilot:
 
 - Rx Out Mode = crsf
+- Rx Ser Baudrate = 57600
 - Rx Ser Link Mode = mavlink
-- Rx Snd RadioStat = w txbuf
-
-
-## OpenTx and Yaapu Telemetry App
+- Rx Snd RadioStat:
+    - mLRS version >= 0.3.13 = ardu_1
```

Comment:
ah, sorry, forgot: v0.3.13? 
dito next line

Reply:
Updated both to 'v0.3.13'